File: app/src/main/java/cc/smartcash/wallet/Models/SmartApiLast.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.io.Serializable;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SmartApiLast implements Serializable {
    private String id;

    private String created;

    public String getId() {
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCreated() {
        return this.created;
    }

    public void setCreated(String created) {
        this.created = created;
    }
}

File: app/src/main/java/cc/smartcash/wallet/Models/SmartApiLast.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
data class SmartApiLast(
        var id: String? = null,
        var created: String? = null
) : Serializable
